### PR TITLE
Updated roku info for client codec support

### PR DESCRIPTION
**Pull request by neilsb**
Information from https://developer.roku.com/en-gb/docs/specs/streaming.md

### CODE REVIEW

File: general/clients/codec-support.md [hunk 1]
```diff
@@ -74,6 +74,9 @@ If the audio codec is unsupported or incompatible (such as playing a 5.1 channel
 
 <sup>5</sup>Safari only supports opus in `.caf` files
 
+<sup>6</sup>Supported via passthrough on all devioces.  Native support for AC3 &amp E-AC3 on Roku TVs &amp; Ultra
```
Comment (neilsb) [inline general/clients/codec-support.md:77]:
```suggestion
<sup>6</sup>Supported via passthrough on all devices.  Native support for AC3 &amp E-AC3 on Roku TVs &amp; Ultra
```

Fix spelling

File: general/clients/codec-support.md [hunk 2]
```diff
@@ -74,6 +74,9 @@ If the audio codec is unsupported or incompatible (such as playing a 5.1 channel
 
 <sup>5</sup>Safari only supports opus in `.caf` files
 
+<sup>6</sup>Supported via passthrough on all devices.  Native support for AC3 &amp E-AC3 on Roku TVs &amp; Ultra
```
Comment (neilsb) [inline general/clients/codec-support.md:77]: Ampersand not showing correctly

**Comment by dkanada** [comment]
@neilsb linting errors are still showing up :p

File: general/clients/codec-support.md [hunk 3]
```diff
@@ -74,6 +74,9 @@ If the audio codec is unsupported or incompatible (such as playing a 5.1 channel
 
 <sup>5</sup>Safari only supports opus in `.caf` files
 
+<sup>6</sup>Supported via passthrough on all devices.  Native support for AC3 &amp; E-AC3 on Roku TVs &amp; Ultra
+
+
```
Comment (neilsb) [inline general/clients/codec-support.md:79]:
```suggestion
```

**Comment by neilsb** [comment]
Rebased

File: general/clients/codec-support.md [hunk 4]
```diff
@@ -74,6 +74,8 @@ If the audio codec is unsupported or incompatible (such as playing a 5.1 channel
 
 <sup>5</sup>Safari only supports opus in `.caf` files
 
+<sup>6</sup>Supported via passthrough on all devices.  Native support for AC3 &amp; E-AC3 on Roku TVs &amp; Ultra
```
Comment (dkanada) [inline general/clients/codec-support.md:77]:
```suggestion
<sup>6</sup>Supported via passthrough on all devices. Native support for AC3 & E-AC3 on Roku TVs & Ultra.
```

**Review by dkanada — APPROVED** [review]

**Pull request merged**